scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/AnnotatedOpDependencyMember.java

```diff
@@ -74,13 +99,12 @@ public boolean isAdaptable() {
 
 	@Override
 	public String getKey() {
-		return key;
+		return keyGenerator.get();
```

Comment:
Cache it? Otherwise, what if it's expensive to call `get()` every time? What was our actual use case for using `Supplier` here?

Reply:
Fixed in  b54956b3